components.md

```diff
@@ -4,11 +4,68 @@ title: Components
 permalink: /components/
 ---
 
+## PELUX Yocto layers
+
+The entry point to the PELUX Yocto/OE build system is the
+[pelux-manifests](https://github.com/Pelagicore/pelux-manifests) repository. It
+contains the information on the layers that are used for different flavours of
+the project build.
+
+OSS layers:
+
+* [poky](https://git.yoctoproject.org/cgit.cgi/poky) - Poky Build Tool and
+  Metadata
+* [meta-openembedded](http://git.openembedded.org/meta-openembedded/) -
+  Collection of OpenEmbedded layers
+* [meta-ivi](https://github.com/GENIVI/meta-ivi) - Layer for InVehicle
+  Infotainment software
+* [meta-swupdate](https://github.com/sbabic/meta-swupdate) - Recipes to support
+  deployment of Yocto's images based on swupdate project
+* [meta-virtualization](https://git.yoctoproject.org/cgit/cgit.cgi/meta-virtualization/) -
+  Layer enabling hypervisor, virtualization tool stack, and cloud support
+* [meta-qt5](http://code.qt.io/cgit/yocto/meta-qt5.git/) - Vendor branches of
+  [https://github.com/meta-qt5/meta-qt5](https://github.com/meta-qt5/meta-qt5)
+* [meta-boot2qt](http://code.qt.io/cgit/yocto/meta-boot2qt.git/) - Layer for
+  Boot2Qt
+* [meta-arp](https://github.com/Pelagicore/meta-arp/) - Layer providing BSP
+  for Automotive Reference Platform
+* [meta-intel](https://git.yoctoproject.org/cgit.cgi/meta-intel/) - Layer
+  containing Intel hardware support metadata
+* [meta-raspberrypi](https://git.yoctoproject.org/cgit.cgi/meta-raspberrypi/) -
+  Hardware specific BSP overlay for the RaspberryPi device
+
+Layers maintained by the PELUX project:
+
+* [meta-pelux](https://github.com/pelagicore/meta-pelux) - Contains the
+  reference integration platform and the PELUX image recipes.
+* [meta-bistro](https://github.com/pelagicore/meta-bistro) - Adds IVI specific
+  components and additions to relevant OSS components.
+* [meta-template](https://github.com/pelagicore/meta-template) - Contains template
+  recipes and components that can be used as examples when starting your own R&D
+  work.
+
+## Development Handbook
+
+The [Development Handbook](//pelux.io/software-factory/) contains most of
+the documentation around how to use PELUX and how to set up a new project.
+
+## SoftwareContainer
+
+The [SoftwareContainer](https://github.com/Pelagicore/softwarecontainer) is a
+framework to manage and contain applications created and vetted by third party
+developers in an automotive setting. A launcher (an UI for example) sends a
+signal to the SoftwareContainer to start a new Container. It can then configure
+the Container using a documented [JSON](https://www.json.org/) format and can
+launch processes inside the Container. This can then be used to contain
```

Comment:
```suggestion
 launch processes inside the Container. This can then be used to isolate
```

Reply:
Text copied from [softwarecontainer](https://github.com/Pelagicore/softwarecontainer) so we need to update the text there too, and I agree that the text is weird. Will fix